test/unittests/test_command_line_processing.py

```diff
@@ -148,7 +148,8 @@ def test_config_success_returns_EX_OK(lobotomize, local_unittest_config, session
 
 
 def test_dump_success_returns_EX_OK(lobotomize, local_unittest_config, session_file):
-    cosmic_ray.cli.main(['init', local_unittest_config, session_file])
+    errcode = cosmic_ray.cli.main(['init', local_unittest_config, session_file])
+    assert errcode == 0
```

Comment:
Should this also be os.EX_OK, for consistency.

Reply:
Definitely!